### ISSUE TITLE
model properties allows non-conforming notation

### ISSUE THREAD

**Issue by joka123** [FriendsOfTYPO3/extension_builder#723]
it is possible to enter not allowed characters like '-,% ...' in "Property names

TYPO3 12

**Extension Builder Version:**
v12.0.0-alpha.6

**Comment by joka123** [comment]
i find some more little issues like input type string is set in tca to 'size' => 0,
you are interested in such things? i won't spam you

**Comment by PKuhlmay** [comment]
Yes just open new issues. I can still put them together. Thank you.

I haven't done much validation for the GUI yet, so there might me a lot more invalid inputs.

**Comment by DavidBruchmann** [comment]
the property names have indeed limits concerning allowed signs. So `%` is certainly reasonable blocked.

**Comment by PKuhlmay** [comment]
This has been fixed in the current Lit JS GUI — property name fields use  validation that filters out invalid characters while typing.
